pkg/errdefs/errors.go

```diff
@@ -40,3 +40,12 @@ func isErrConnectionRefused(err error) bool {
 func NeedsRetryWithHTTP(err error) bool {
 	return err != nil && (isErrHTTPResponseToHTTPSClient(err) || isErrConnectionRefused(err))
 }
+
+func isErrDifferentFsVersion(err error) bool {
```

Comment:
Not only the fs version, how about renaming it `isErrInconsistentNydusLayer`.

Reply:
fine